frontend/src/components/explorer/mapViewer/SidebarDataPanels.vue

```diff
@@ -290,6 +301,20 @@ export default {
       this.showFullReactionListMissing = false;
       this.showFullReactionListMap = false;
     },
+    toggleSelectionCardContent() {
```

Comment:
This function could take in an optional variable to also achieve the same functionality as `openSelectionCardContent` and `closeSelectionCardContent`.

Reply:
I think this is where I personally prefer the clarity for the developers. For example when just looking at where the methods are being called (in markup or from another component), it's straightforward to know the intent without looking into the method definitions. It's not too much difference either way for this case though but that's my general stance. What do you think @inghylt and @nanjiangshu? I don't want to enforce having too many methods if you think it's unnecessary too.

Reply:
I would say that in general, I prefer having multiple dense and straightforward methods. It's kind of context dependent though, e.g. in this particular case, it seems like the methods can be combined without loosing straightforwardness. But I would also be fine leaving it as it is right now

Reply:
I just want to add that by definition, toggle means switching between two states so it doesn't 100% make sense for it to take an optional variable. By not taking a variable and using the three functions, the binary intent of each function is directly implied.

If this had to be one function maybe it should be renamed to something like `setSelectionCardContentVisibility` (probably needs a better name since visibility feels awkward).

Anyways, while this is worth a discussion, I think it is too minor of an issue to hold up this PR. So I will resolve this conversation without fixing it but I would love to discuss more about coding standards and practices.